Cargo.toml

```diff
@@ -1,6 +1,5 @@
 [package]
 name = "rustup"
-version = "1.27.1"
```

Comment:
This is the version of the Rustup package, not the version of the compiler...

Reply:
Oh sorry! I got `version` and `rust-version` mixed up!